payments/management/commands/update_discount_code.py

```diff
@@ -0,0 +1,130 @@
+from datetime import datetime
+
+import pytz
+from django.core.management import BaseCommand
+
+from payments.api import update_discount_codes
+from unified_ecommerce import settings
+
+
+class Command(BaseCommand):
+    """
+    Updates one or multiple discount codes using the Discount IDs.
+    example usage of this command:
+    python manage.py update_discount_code 1 2 3 --expires 2023-01-01 --amount 10 \
+    --discount-type dollars-off --payment-type marketing
+    """
+
+    help = "Updates one or multiple discount codes using the Discount IDs."
+
+    def add_arguments(self, parser) -> None:
+        parser.add_argument(
```

Comment:
Need to use the code instead. Additionally, we also need to be able to specify that the codes are part of a batch so we can modify the whole batch.

Reply:
We discussed this in slack so marking as resolved